lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/TaskManager.cs

```diff
@@ -322,6 +344,17 @@ internal bool AreAllTasksCompleted()
         }
 
         /// <summary>
+        /// Either master is completed or all the tasks are completed.
+        /// In fact AreAllTasksCompleted contains IsmasterCOmpleted
+        /// Put it in a separate method so that We can update the logic for job done when needed
+        /// </summary>
+        /// <returns></returns>
+        internal bool IsJobDone()
+        {
+            return IsMasterTaskCompleted() || AreAllTasksCompleted();
```

Comment:
And maybe remove `AreAllTasksCompleted` method altogether, since it will be unused.